src/logic/create.ts

```diff
@@ -65,8 +66,9 @@ export async function createProfile(
   );
 
   if (created) {
+    console.log(logger);
     logger.success('createProfile', profile.withVersion(versionStr), filePath);
-    if (superJson && superJsonPath) {
+    if (superJson && superJsonPath !== undefined) {
```

Comment:
superJson !== undefined 🤔

Reply:
why didn't the linter catch it? :thinking: